yaptide/batch/batch_methods.py

```diff
@@ -224,22 +225,42 @@ def prepare_script_files(payload_dict: dict, job_dir: str, sim_id: int, update_k
 
     backend_url = os.environ.get("BACKEND_EXTERNAL_URL", "")
 
-    submit_script = SUBMIT_SHIELDHIT.format(array_options=array_options,
+    logging.info(payload_dict)
+
+    if payload_dict['sim_type'] == SimulationType.FLUKA.value:
+        submit_script = SUBMIT_FLUKA.format(array_options=array_options,
```

Comment:
if all the functions have the same parameter, then why not assign the template, like:
```
if payload_dict['sim_type'] == SimulationType.FLUKA.value:
   submit_template  = SUBMIT_FLUKA
elif payload_dict['sim_type'] == SimulationType.SHIELDHIT.value:
   submit_template  = SUBMIT_SHIELDHIT
submit_script = submit_template.format(array_options=array_options,
                                                collect_options=collect_options,
                                                root_dir=job_dir,
                                                n_tasks=str(payload_dict["ntasks"]),
                                                convertmc_version=pymchelper.__version__)
```

then the code would be much shorter